pandas/tests/test_resample.py

```diff
@@ -2252,12 +2278,17 @@ def test_selection(self):
                           index=pd.MultiIndex.from_arrays([
                               np.arange(len(index), dtype=np.int64),
                               index], names=['v', 'd']))
-
-        with pytest.raises(NotImplementedError):
-            df.resample('2D', on='date')
-
-        with pytest.raises(NotImplementedError):
-            df.resample('2D', level='d')
+        for freq in ['H', '12H', '2D', 'W']:
```

Comment:
not sure if I understood correctly - pushed 48e324f as an example, are you thinking along those lines of using pytest fixtures/mark.parametrize decorators?